tajo-docs/src/main/sphinx/index/how_to_use.rst

```diff
@@ -48,17 +49,17 @@ If the index is created successfully, you can see the information about that ind
 
 For more information about index creation, please refer to the above links.
 
--------------------------------------
+-----------------------------
 2. Enable/disable index scans
--------------------------------------
+-----------------------------
 
-When an index is successfully created, you must enable the index scan feature as follows:
+When an index is successfully created, you must enable index scan as follows:
```

Comment:
This section is related to the previous section ```Create index```. So, starting with ```even though``` looks not good I think. 
How about ```Reading data using index is disabled by default.
So, exploiting the created index, you need a further step, enabling 'index scan' as following:```?